packages/kit/src/runtime/server/page/render.js

```diff
@@ -394,7 +412,12 @@ export async function render_response({
 				hydrate.push(`status: ${status}`);
 			}
 
-			if (options.embedded) {
+			if (manifest._.client.routes) {
+				if (event.route.id) {
+					const route = create_stringified_csr_server_route(event.route.id, event.url, manifest);
```

Comment:
Cosmetic change but we take pains to make the generated code readable elsewhere, and this helps

```suggestion
					const route = create_stringified_csr_server_route(event.route.id, event.url, manifest).replaceAll('\n', '\n\t\t\t\t\t\t\t');
```